iris/response.go

```diff
@@ -29,7 +29,7 @@ type BeneficiaryBankResponse struct {
 type CreatePayoutResponse struct {
 	Payouts      []CreatePayoutDetailResponse `json:"payouts"`
 	ErrorMessage string                       `json:"error_message"`
-	Errors       []string                     `json:"errors"`
+	Errors       interface{}                  `json:"errors"`
```

Comment:
it probably good idea to add test-case for this scenario too, to make sure no error is thrown, when this struct is called

Reply:
Sure, will create the test for this one

Reply:
Done, added the CreatePayoutFail test